credsweeper/filters/value_jfrog_token_check.py

```diff
@@ -0,0 +1,37 @@
+import contextlib
+import re
+
+from credsweeper.common.constants import ASCII
+from credsweeper.config import Config
+from credsweeper.credentials import LineData
+from credsweeper.file_handler.analysis_target import AnalysisTarget
+from credsweeper.filters import Filter
+from credsweeper.utils import Util
+
+
+class ValueJfrogTokenCheck(Filter):
+    """Check that candidate have a known structure JFROG token"""
+
+    def __init__(self, config: Config = None) -> None:
+        # reftkn:01:0123456789:abcdefGhijklmnoPqrstuVwxyz0
+        self._pattern = re.compile(r"reftkn:\d+:\d+:[\w_/+-]+")
```

Comment:
Can we sure it is refresh token of Jfrog only with this pattern?
I don't know well, but `reftkn` can be used other cases also..
If you know about the Jfrog token please let me know.

Reply:
I assume, the decoded data matched for the pattern is the token type. At least, the provider encode their token with the approach. 
Unfortunately, there is no info what structure inside. Just my experience.
https://jfrog.com/help/r/platform-api-key-deprecation-and-the-new-reference-tokens/what-is-a-reference-token

Reply:
It seems reference token.. not refresh token.
I think refresh token is more close with below link.
https://jfrog.com/help/r/jfrog-rest-apis/refresh-token

As I understood, refresh token can refresh the access token even if i don't have access token.
So yes, it is credential, but there is no clue of this refresh token's structure..
Could you check this more?